tests/test_device.py

```diff
@@ -78,24 +79,21 @@ def test_duplicated_actuators(self, device):
                 SensorActuator(dpi=1000)
             ]
 
-    def test_mouse_report(self, device, libevdev_event_nodes):
+    def test_mouse_report(self, device, event_data):
         '''
         Test single HID report
         '''
         x = y = 5
 
         expected = EventData(x, y)
 
-        def callback(device):
-            nonlocal expected
-            device.send_hid_action(expected)
+        device.send_hid_action(expected)
+        time.sleep(0.1)
```

Comment:
You need to add a comment explaining why you need the sleep here, especially given that `send_hid_action` should be synchronous.